composer.json

```diff
@@ -14,7 +14,9 @@
         "aura/di": "dev-develop-2",
         "bodom78/kohana-imagefly": "dev-master",
         "league/oauth2-server": "~3.2.4",
-        "league/flysystem": "0.4.*",
+        "league/flysystem-aws-s3-v3": "1.0.4",
```

Comment:
That would be my pref if it's not horrorific for us. Theoretically with the ansible deployment scripts we get a degree of control over what should be installed on system AND I don't think in the age of cheap VMs it's unreasonable to force the latest version of everything - where appropriate